examples/tutorials/doc_examples/examples/observe_different_events.py

```diff
@@ -37,6 +38,7 @@ def notify_scores_content_change(self, event):
             "(Event type: {event.__class__.__name__})".format(event=event)
         )
 
+
```

Comment:
These examples are being included in the user manual literally. I think I deliberately reduced the newlines so that they don't take up so much space on the doc. That was entirely subjective so please feel free to ignore.

Reply:
Yes, I guessed, but I don't feel bad about having the examples in the doc follow flake8 style. I _would_ like these examples to be included in the style check generally, so the alternative is to add a per-file-ignore for this particular error in the `setup.cfg` (adding a `# noqa` comment doesn't seem like a good option for something appearing in the docs). It's a tradeoff, but this seems like the most straightforward solution.

Reply:
Yep. I don't care about the extra white space on the doc that much. I just thought I'd mention. This is fine.